packages/assets-controllers/src/NftController.ts

```diff
@@ -2060,6 +2060,17 @@ export class NftController extends BaseController<
       });
     }
   }
+
+  /**
+   * Reset the controller state to the initial state.
+   */
+  resetState() {
+    this.update((state) => {
+      state.allNftContracts = {};
+      state.allNfts = {};
+      state.ignoredNfts = [];
+    });
+  }
```

Comment:
The initial state is the state that the controller is initialized with, but aren't we resetting it to the default state instead? If that's true then what do you think about reusing the default state function, and returning what it returns? (See the [Immer docs](https://immerjs.github.io/immer/return) for more on why this is possible.)

``` suggestion
  /**
   * Reset the controller state to the default state.
   */
  resetState() {
    this.update(() => {
      return getDefaultNftControllerState();
    });
  }